install_template/README.md

```diff
@@ -225,3 +225,44 @@ In a handful of situations it is useful to employ conditionals to include or mod
 Be wary of over-using these; prefer inheritance structures that push this content down to sufficiently-specific "leaf" templates instead. As a general rule, use conditionals only when the test is much, much simpler than the alternative inheritance structure, and be willing to abandon it when (over time) that simplicity is lost.
 
 In particular, avoid the trap of setting a flag in a leaf template and then checking it in a base template: this separates the context from the decision, making it extremely difficult to judge when the test has lost its value.
+
+#### Adding a new platform
+
+1. Modify **config.yaml**. For example, when adding the RHEL 9 platform, the following entries were made to each product:
```

Comment:
I'd like to see some precision here on what you add:

- name:
String: Name of the platform (or platforms) (Is there a reference set of names we work with? When adding a new version are there root names we should use?)
  arch: 
String: CPU Architecture (again, reference list of architecture names anywhere? What if its a new architecture (coming soon for example, aarch64)
  supported-versions:
What is this? A list? What precision of version number, what delimiter?

Reply:
Good suggestions, Dj. Nidhi, Dave, and I have discussed this a bit and we feel we should limit the scope in this PR, but try to add more information to the readme in a separate PR. Nidhi and I both thought that the general topic could be added to the agenda for your weekly meeting with Nidhi.